packages/package.go

```diff
@@ -130,20 +134,22 @@ func LoadPackageFromPath(path string) (*Package, error) {
 	if err = yaml.Unmarshal(d, pkg); err != nil {
 		return pkg, err
 	}
+
+	if pkg.Name == "" {
+		pkg.Name = filepath.Base(path)
```

Comment:
In which case did you find out that `pkg.Name` was empty?
I am seeing that `NewPackageFromImage` is always setting it by default, hence it should always be populated after install

Reply:
I noticed pkg.Name was empty after loading a package via packages.LoadPackageFromPath.  I only noticed this when deep comparing a loaded  package vs one created from image inspection.

Reply:
nit: we could use the same defaulting method as above 
```
pkg := &Package{
			WorkingDir: DefaultWorkingDir,
			Name: filepath.Base(path),
		}
```

Reply:
Made this change